### ISSUE TITLE
Scatter Plot uses `chart.scatter.onlyCircles(false);`

### ISSUE THREAD

**Issue by pmeier82** [areski/python-nvd3#93]
The part `chart.scatter.onlyCircles(false);` in the javascript code generated by the scatter plot example is throwing a javascript error: `Uncaught TypeError: undefined is not a function`.

Uncommenting this line cures the problem. Are there any API changes or typos in the examples?

**Comment by areski** [comment]
Make sure you are using >= 1.7 version of NVD3

**Comment by pmeier82** [comment]
I am using:
```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<link href="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.1/nv.d3.css" rel="stylesheet"/>
<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.1/nv.d3.js"></script>
```

which should be up to date.

**Comment by areski** [comment]
sorry my bad, it's actually the contrary onlyCircles has been removed.
The  last version of python-nvd3 should support it https://github.com/areski/python-nvd3/commit/7933c51f379a648fa32dd17de7427bbed66af786#diff-b13a7cae75df2d8705225ca0d3c4c192L89

**Comment by pmeier82** [comment]
Ok, pip resolved python-nvd3 to the 0.13.7 tag per requirements.txt, which has the `onlyCircles` line still in it. Are you going to create a new version/tag anytime soon?

**Comment by areski** [comment]
yes we might have a new version in the following days
